src/main/resources/templates/index.html

```diff
@@ -60,7 +59,7 @@
         </div>
         <div class="container my-2" style="text-align:center" ;>
             <h1>Event List</h1>
-            <table border="1" class="table table-striped table-bordered table-responsive-md">
+            <table style="border: 1px solid #000;" class="table table-striped table-bordered table-responsive-md">
                 <thead>
                     <tr>
                         <th>Event</th>
```

Comment:
border is an outdated html attribute. I replaced it with its CSS alternative

Reply:
Good.  Go one step further and move it to a linked CSS stylesheet.